hasher-matcher-actioner/terraform/migrations/README.md

```diff
@@ -1,8 +1,8 @@
 # What are migrations
 
-Migrations are commands to run when installing a new version of HMA. This includes both a fresh install or an upgrade of an older version. 
+Migrations are commands to run when installing a new version of HMA. 
 
-In both cases, there may be things that can't be done from within terraform, like creating some database entries, or running an hmalib.cli command, etc.
+There may be things that can't be done from within terraform, like creating some database entries, or running an hmalib.cli command, etc.
```

Comment:
There's a missing connecting statement here somewhere like "We need to use scripts because there are certain actions..."

Reply:
yeah, I'll redraft it more.